crates/sui-framework/packages/bridge/sources/message.move

```diff
@@ -191,6 +204,10 @@ module bridge::message {
         token_type: u8,
         amount: u64
     ): BridgeMessage {
```

Comment:
Shouldn't we also validate that the route is valid when creating these msgs?

Reply:
this is just a constructor function for the BridgeMessage struct so not doing any checks here, the message won't be able to get any signature from the validators if the route is invalid anyway, so won't do any harm to the bridge